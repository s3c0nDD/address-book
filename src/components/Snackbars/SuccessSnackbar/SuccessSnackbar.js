import React from 'react';
import PropTypes from 'prop-types';

import CommonSnackbar from '../CommonSnackbar';

const SuccessSnackbar = ({ open, message }) => {
  return (
    <CommonSnackbar
      open={open}
      message={message}
      severity="success"
    />
  )
};

export default SuccessSnackbar;

SuccessSnackbar.propTypes = {
  message: PropTypes.string.isRequired,
  open: PropTypes.bool
};