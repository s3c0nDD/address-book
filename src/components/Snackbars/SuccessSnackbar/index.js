import SuccessSnackbar from './SuccessSnackbar';

export default SuccessSnackbar;