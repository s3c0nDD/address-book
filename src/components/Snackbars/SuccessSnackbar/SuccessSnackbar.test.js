import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';

import SuccessSnackbar from './SuccessSnackbar';

describe('SuccessSnackbar', () => {
  let shallow;
  let component;
  let wrapper;

  beforeAll(() => {
    shallow = createShallow();
  });

  beforeEach(() => {
    component = (
      <SuccessSnackbar
        message="Hello world"
        open={true}
      />
    );
    wrapper = shallow(component);
  });

  it('shallowly renders and matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
