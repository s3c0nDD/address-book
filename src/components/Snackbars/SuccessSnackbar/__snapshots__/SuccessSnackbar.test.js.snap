// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessSnackbar shallowly renders and matches snapshot 1`] = `
<CommonSnackbar
  message="Hello world"
  open={true}
  severity="success"
/>
`;
